outbound/adapter/messages-outbound-enricher.ts

```diff
@@ -0,0 +1,58 @@
+import * as TE from "fp-ts/TaskEither";
+import { flow, pipe } from "fp-ts/lib/function";
+import {
+  MessageModel,
+  RetrievedMessage
+} from "@pagopa/io-functions-commons/dist/src/models/message";
+import { BlobService } from "azure-storage";
+import * as RA from "fp-ts/ReadonlyArray";
+import * as T from "fp-ts/Task";
+import * as E from "fp-ts/Either";
+import { OutboundEnricher } from "../port/outbound-enricher";
+import { failure, success } from "../port/outbound-publisher";
+
+const DEFAULT_THROTTLING = 500;
```

Comment:
Do we need a config variable here that has a fallback on a default instead of constant ?

Reply:
yes, it' better to have it configurable. I moved it in config.ts with the same defalt value.